### PR TITLE
Group outcomes

### PR DESCRIPTION
**Pull request by trey-stafford**

### CODE REVIEW

File: doc/notes/post-workshop-todo.md [hunk 1]
```diff
@@ -46,6 +81,9 @@ _Should this be in the admin space, or the outcomes space?_
 
 * Overall impressions
 * Changes we'd want to make
+    * Symbology exercise: deliverable in addition to qmd files? Screenshot? Discussion post?
```
Comment (MattF-NSIDC) [inline doc/notes/post-workshop-todo.md:84]:
```suggestion
    * Symbology exercise: deliverable in addition to qml files? Screenshot? Discussion post?
```

File: outcomes.md [hunk 2]
```diff
@@ -18,6 +18,33 @@ Our participants engaged in small group discussions on
 [GitHub Discussions](https://github.com/orgs/qgreenland-workshop-2023-researcher/discussions/categories/fair-care).
 
 
+### Explored geospatial data using Jupyter Notebooks
+
+Participants worked together to explore, transform, and visualize geospatial
+data from a variety of sources using [Jupyter Notebooks](https://jupyter.org/).
+
+For exmaple:
```
Comment (trey-stafford) [inline outcomes.md:26]: I just picked out a couple of the better examples and highlighted them here.

File: doc/notes/post-workshop-todo.md [hunk 3]
```diff
@@ -41,6 +81,10 @@ _Should this be in the admin space, or the outcomes space?_
 
 * Overall impressions
 * Changes we'd want to make
+    * Symbology exercise: deliverable in addition to qml files? Screenshot? Discussion post?
+    * More time for exercises, especially on day 1. Maybe fewer topics covered overall?
```
Comment (MattF-NSIDC) [inline doc/notes/post-workshop-todo.md:85]:
```suggestion
    * More time for exercises, especially on day 1. Maybe allocate more time than 9 hours, or cover fewer topics.
```

File: doc/notes/post-workshop-todo.md [hunk 4]
```diff
@@ -41,6 +81,10 @@ _Should this be in the admin space, or the outcomes space?_
...
+    * Should be more clear on file names/structure for git repos. Initially hard to tell which notebooks correspond to each exercise.
+    * Archive workshop materials? E.g., Everything in qgis-data?
```
Comment (MattF-NSIDC) [inline doc/notes/post-workshop-todo.md:87]: Good idea. Too late already I think for our workshop, but hopefully that thought benefits someone else in the future :)

**Pull request merged**